projects/netgrif-components/src/lib/data-fields/text-field/rich-textarea-field/rich-textarea-field.component.html

```diff
@@ -4,7 +4,7 @@
 
     <input type="hidden" [formControl]="formControlRef">
 
-    <div class="border" *ngIf="formControlRef.disabled">
+    <div [ngClass]="{'border': textAreaField.materialAppearance === 'outline'}" *ngIf="formControlRef.disabled">
```

Comment:
constant

Reply:
it is hard to put enum into HTML, you need to create a variable in .ts for that , and I think its not nice to put there "dead" variables for it